sui/src/unit_tests/sui_network.rs

```diff
@@ -3,33 +3,59 @@
 
```

Comment:
The same comments as `faucet/src/test_utils.rs` apply, for the obvious reasons.
One alternative to duplicating this work would be exporting test utils to their own crate (itself imported by dependents under `#[cfg(test)]`.

Reply:
do you mean duplicate  code? if so, I agree